mapbox/dr-ui
# [Info arch] Allow appending tabs, including links to external pages, to nav tabs

This PR adds an option for appending an arbitrary list of tabs to the nav accordion. This includes external links, to which `NavigationAccordion` will automatically add a "share" icon to denote that the link is to an external page.

<img width="257" alt="image" src="https://user-images.githubusercontent.com/2365503/98870313-c366e100-2438-11eb-8b82-1d780f328bbe.png">


## How to test

- Pull down this branch
- `npm run start-docs` to build the dr-ui docs site
- Observe the external links in the navigation accordion
- [optionally] try adding another page to the `addPages` variable in the docs site's `batfish.config.js` file

## QA checklist



- [x] Accessibility score in [Chrome DevTools Audit/Lighthouse](https://developers.google.com/web/tools/lighthouse#devtools) is 96% with Lighthouse version: `6.2.0`, specifically for batfish helpers guide.

Open the test cases app locally on:
